platform/broadcom/sonic-platform-modules-accton/as7326-56x/sonic_platform/chassis.py

```diff
@@ -8,58 +8,58 @@
 
 try:
     import sys
-    import time
     from sonic_platform_pddf_base.pddf_chassis import PddfChassis
+    from .event import SfpEvent
+    from .helper import APIHelper
 except ImportError as e:
     raise ImportError(str(e) + "- required module not found")
 
+NUM_COMPONENT = 6
+HOST_REBOOT_CAUSE_PATH = "/host/reboot-cause/"
+PMON_REBOOT_CAUSE_PATH = "/usr/share/sonic/platform/api_files/reboot-cause/"
+REBOOT_CAUSE_FILE = "reboot-cause.txt"
 
 class Chassis(PddfChassis):
     """
     PDDF Platform-specific Chassis class
     """
 
+    SYSLED_DEV_NAME = "DIAG_LED"
```

Comment:
So chassis set/get status_led will lead to setting of DIAG led ?
This platform doesn't have SYS_LED ?

Reply:
We  have DIAG_LED and LOC_LED, and don't have SYS_LED.  So use DIAG_LED instead of SYS_LED.